add helper callGetIteratorRange

fdecampredon opened the pull request:
It is a simple helper function that allows to retrieve a range of element from an iterator, used in https://github.com/fdecampredon/es6-destructuring

eventualbuddha commented:
Partly as a result of this PR I've updated how ast-util manages the helpers it injects. Please see [getIterator.js](https://github.com/square/ast-util/blob/master/helpers/getIterator.js) and the corresponding [Makefile entry](https://github.com/square/ast-util/blob/master/Makefile#L13-L21) for an example of how to do this.

fdecampredon committed:
add helper callGetIteratorRange

It is a simple helper function that allows to retrieve a range of
element from an iterator

fdecampredon commented:
@eventualbuddha I have adapted the helper to the new build system
